www/source/tutorials/getting-started-process-build.html.md

```diff
@@ -13,14 +13,14 @@ For more information on how to natively run Habitat services or how to run servi
 
 To show the portability of Habitat, you will export and run a Habitat service from within a Docker container through the following steps:
 
-1. If you are not in the studio environment, change over to your plan directory and enter the studio.
+1. If you are not in the studio environment, change over to your plan directory and enter the studio.  (Remember to replace `myorigin` with the name of the origin key you created during `hab setup`. For the examples in this tutorial, it will be set to "myorigin")
 
-        cd ~/Plans/mytutorialapp
+        cd ~/plans/mytutorialapp
         hab studio -k myorigin enter
 
 2. Build your mytutorialapp package.
 
-        [1][default:/src:0]# build
+        [1][default:/src:0]# build mytutorialapp
```

Comment:
When you `cd` in the step above and then enter the studio, you don't need to specify the name of the package because you're already inside the mytutorialapp directory. Doing so will cause an error.